File: internal/model/skill.go
package model

import "time"

// Skill represents an installed skill in local storage.
type Skill struct {
	ID          string    `json:"id"`
	Name        string    `json:"name"`
	Description string    `json:"description"`
	Author      string    `json:"author"`
	Version     string    `json:"version"`
	Tags        []string  `json:"tags"`
	Agents      []string  `json:"agents"`
	Content     string    `json:"content"`
	LocalPath   string    `json:"localPath"`
	SourceURL   string    `json:"sourceUrl"`
	InstalledAt time.Time `json:"installedAt"`
	UpdatedAt   time.Time `json:"updatedAt"`
	IsManaged   bool      `json:"isManaged"`
}

File: frontend/src/types/skill.ts
// Skill 表示已安装的技能
export interface Skill {
  id: string
  name: string
  description: string
  author: string
  version: string
  tags: string[]
  agents: string[]
  content: string
  localPath: string
  sourceUrl: string
  installedAt: string
  updatedAt: string
}
